ApollosProject/apollos-prototype
# Fixes safe area layout provider

## DESCRIPTION

### What does this PR do, or why is it needed?

Existing layout provider was returning 0 for the bottom offset under every condition. I realized it would be impossible to fix our issues with the MediaPlayer spacer unless we fixed the layout provider, and I couldn't find any good solutions, so I just used an off the shelf one mentioned by `react-navigation` 😂 

### What design trade-offs/decisions were made?

### How do I test this PR?

Check the height of the tab bar with the mini media player open, and closed. It should have tasteful margins under both conditions.

### What automated tests did you write?

n/a

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
